simulator/src/main/java/com/hazelcast/simulator/test/annotations/InjectMetronome.java

```diff
@@ -15,33 +15,15 @@
  */
 package com.hazelcast.simulator.test.annotations;
 
-import com.hazelcast.simulator.worker.metronome.MetronomeType;
-
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import static com.hazelcast.simulator.worker.metronome.MetronomeType.NOP;
-
 /**
  * Annotates {@link com.hazelcast.simulator.worker.metronome.Metronome} fields.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface InjectMetronome {
-
-    /**
-     * Defines the {@link com.hazelcast.simulator.worker.metronome.Metronome} interval in milliseconds.
-     *
-     * @return the {@link com.hazelcast.simulator.worker.metronome.Metronome} interval in milliseconds
-     */
-    int intervalMillis() default 0;
```

Comment:
The metronome should not be configured in the test because it will hardcode the settings. Metronome is configured from the outside; so these properties need to go.